lab_2_keywords_cooccurrence/main.py

```diff
@@ -161,4 +284,7 @@ def load_stop_words(path: Path) -> Optional[Mapping[str, Sequence[str]]]:
     :param path: path to the file with stop word lists
     :return: a dictionary containing the language names and corresponding stop word lists
     """
-    pass
+    if not isinstance(path, Path):
+        return None
+    with open(path, 'r', encoding='utf-8') as stop_words:
+        return dict(json.load(stop_words))
```

Comment:
why do we have to convert loaded content to dict?

Reply:
so we can get the keys, which contain stop words, by writing the needed value